src/models/document/base-document-content.ts

```diff
@@ -187,6 +189,14 @@ export const BaseDocumentContentModel = types
     };
   })
   .views(self => ({
+    getRowHeight(rowId: string) {
+      const row = self.getRow(rowId);
+      if (!row) return 0;
+      if (row.isSectionHeader) return kSectionHeaderHeight;
+      if (self.isPlaceholderRow(row)) return kPlaceholderTileDefaultHeight;
+      // NOTE: This may not be accurate for rows with tiles that rely on the getContentHeight tileAPI function.
+      return row.height;
+    },
```

Comment:
This could probably be mostly moved into the row model as a computed value (cached) then `get height()` below would be a bit more efficient.